packages/backend/src/controllers/tasks/tasks.util.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@nestjs/common';
+import { randomInt } from 'crypto';
+import { TaskDocument } from 'src/db/task/task.schema';
+
+@Injectable()
+export class TaskUtil {
+  selectRandomUser(pool) {
+    return pool[randomInt(0, pool.length)];
+  }
+
+  checkRecurrence(tasks: TaskDocument[]) {
+    const result = [];
+
+    tasks.forEach((task) => {
+      //Need to multiply interval by 1000 to increment the date by seconds.
+      const currentDueDate = new Date(
+        task.dueDate.getTime() + 1000 * task.interval,
```

Comment:
task.interval should be in seconds
```suggestion
        task.dueDate.getTime() + task.interval,
```

Reply:
oh this is because getTime is in milliseconds. the comment is wrong